.github/scripts/trymerge.py

```diff
@@ -931,7 +931,7 @@ def merge(pr_num: int, repo: GitRepo,
     initial_commit_sha = pr.last_commit()['oid']
     if force:
```

Comment:
```suggestion
    # do not wait for any pending signals if PR is closed as part of co-development process
    if force or can_skip_internal_checks(pr, comment_id):
```

Reply:
Would we ever run into scenario where PR has failing checks that were not covered by internal jobs?

Reply:
Than it should not have been landed (i.e. person overrode those errors, which is similar to `--force` option)